### DISCUSSION TITLE
Improving Bookmark list

### DISCUSSION THREAD

**Discussion by dineshsrd** [dailydotdev/daily#964] [Feature Requests]
Some update to the feature in [this](https://github.com/dailydotdev/daily/discussions/520) thread
- Sharing the bookmarks folder, either all the bookmarks or specific few

If you plan to bring paid plans in future this could be a paid feature.

**Comment by idoshamun** [comment]
Hi @dineshsrd,

Thanks a lot for the suggestion. Let's keep the discussion in the original thread #520
